jenkins-jobs/job-dsl/connector_all_tests.groovy

```diff
@@ -0,0 +1,37 @@
+pipelineJob('connector-all-tests') {
+    displayName('Debezium All Connectors Test')
+    description('Executes tests for all connectors')
+
+    parameters {
+        booleanParam('DB2_TEST', true, 'Run DB2 Tests')
+        booleanParam('MONGODB_TEST', true, 'Run MongoDB Tests')
+        booleanParam('MYSQL_TEST', true, 'Run MySQL Tests')
+        booleanParam('ORACLE_TEST', true, 'Run Oracle Tests')
+        booleanParam('POSTGRESQL_TEST', true, 'Run PostgreSQL Tests')
+        booleanParam('SQLSERVER_TEST', true, 'Run SQL Server Tests')
+
+        stringParam('REPOSITORY_CORE', 'https://github.com/debezium/debezium', 'Repository from which Debezium is built')
+        stringParam('BRANCH_CORE', 'main', 'A branch/tag from which Debezium is built')
+
+        // db2 specific
+        stringParam('REPOSITORY_DB2', 'https://github.com/debezium/debezium-connector-db2', 'Repository from which DB2 connector is built')
+        stringParam('BRANCH_DB2', 'main', 'A branch/tag from which Debezium is built for DB2 connector')
+
+        stringParam('SOURCE_URL', "", "URL to productised sources")
+        booleanParam('PRODUCT_BUILD', false, 'Is this a productised build?')
+
+        //oracle specific
+        stringParam('MAIL_TO', 'debezim-qe@redhat.com')
```

Comment:
Actually this should be a general parameter. If some jobs don't have it, we should definitively add it :)